.github/workflows/static.yaml

```
@@ -141,9 +141,8 @@ jobs:
         name: Copy binary
         if: ${{ !fromJson(needs.prepare.outputs.push) }}
         run: |
-          image=$(jq -r '."static-builder"."containerimage.config.digest"' <<< "${METADATA}")
-          container=$(docker create --platform=${{ matrix.platform }} --name static-builder "${image}")
-          docker cp "${container}:/go/src/app/dist/${BINARY}" "${BINARY}"
+          tag=$(jq -cr '.target."static-builder".tags | first' <<< "${METADATA}")
+          docker cp "$(docker create --platform=${{ matrix.platform }} --name static-builder "${tag}"):/go/src/app/dist/${BINARY}" "${BINARY}"" ; docker rm static-builder
```

Since we are naming the docker container, I think we can refactor this a bit to make it simpler:

```suggestion
          docker create --platform=${{ matrix.platform }} --name static-builder "${tag}"
          docker cp "static-builder:/go/src/app/dist/${BINARY}" "${BINARY}""
          docker rm static-builder
```